### PR TITLE
Bunch of changes with the form

### PR DESCRIPTION
**Pull request by jaleszek**
@RuslanHamidullin please review carefully, it's my first commits around that.

Moreover:

 I've changed the `Country` field to be `County` in one of the questions for Trade. If I understand the problem correctly now all forms on servers which has the `principal_address_country` instead of county would be invalid ?
so I need to update them all after that

### CODE REVIEW

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def innovation_step1
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
-          context "<p>Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.</p>"
+          context "<p>Organisations that began trading after 01/10/2012 aren't eligible for this award.</p>"
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [108/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 2]
```diff
@@ -58,7 +58,7 @@ def trade_step1
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
-          context "<p>Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.</p>"
+          context "<p>Organisations that began trading after 01/10/2012 aren't eligible for this award.</p>"
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [108/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 3]
```diff
@@ -56,16 +56,14 @@ def innovation_step1
           style "small"
         end
 
-        # TODO: Hardcoded date
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
-          context "<p>Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.</p>"
-          date_max "01/10/2012"
+          context "<p>Organisations that began trading after #{AwardYear.start_trading_moment} aren't eligible for this award.</p>"
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [131/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 4]
```diff
@@ -56,16 +56,14 @@ def innovation_step1
...
+          date_max AwardYear.start_trading_moment
         end
 
-        # TODO: Hardcoded date
-        options :queen_award_holder, "Are you a current Queen's Award holder <span class='todo-placeholder'>(2010-2014)</span>?" do
+        options :queen_award_holder, "Are you a current Queen's Award holder (#{AwardYear.award_holder_range})?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [115/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 5]
```diff
@@ -54,16 +54,14 @@ def trade_step1
           style "small"
         end
 
-        # TODO: Hardcoded date
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
-          context "<p>Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.</p>"
-          date_max "01/10/2012"
+          context "<p>Organisations that began trading after #{AwardYear.start_trading_moment} aren't eligible for this award.</p>"
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [131/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 6]
```diff
@@ -54,16 +54,14 @@ def trade_step1
...
+          date_max AwardYear.start_trading_moment
         end
 
-        # TODO: Hardcoded date
-        options :queen_award_holder, "Are you a current Queen's Award holder <span class='todo-placeholder'>(2010-2014)</span>?" do
+        options :queen_award_holder, "Are you a current Queen's Award holder (#{AwardYear.award_holder_range})?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [115/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb [hunk 7]
```diff
@@ -59,23 +59,21 @@ def development_step1
...
         end
 
-        # TODO: Hardcoded date
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
           context %(
             <p>
-              Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.
+              Organisations that began trading after #{AwardYear.start_trading_moment} aren't eligible for this award.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [118/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb [hunk 8]
```diff
@@ -59,23 +59,21 @@ def development_step1
...
-          date_max "01/10/2012"
+          date_max AwardYear.start_trading_moment
         end
 
         header :business_awards_header, "Business awards" do
         end
 
-        # TODO: Hardcoded date
-        options :queen_award_holder, "Are you a current Queen's Award holder <span class='todo-placeholder'>(2010-2014)</span>?" do
+        options :queen_award_holder, "Are you a current Queen's Award holder (#{AwardYear.award_holder_range})?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [115/100]

**Comment by RuslanHamidullin** [comment]
@jaleszek looks good, but some notes:
1) You need to setup principal_address sub_fields here (as default):
https://github.com/bitzesty/qae/blob/master/lib/qae_form_builder/address_question.rb#L46
instead of in app/forms
2) As you changed name of data field, so need to add simple rake task in order to make existing forms valid again on 4 servers after deploying this progress

**Comment by jaleszek** [comment]
@RuslanHamidullin 
```
1) You need to setup principal_address sub_fields here (as default):
https://github.com/bitzesty/qae/blob/master/lib/qae_form_builder/address_question.rb#L46
instead of in app/forms
```
Are you sure ? 

```
      if sub_fields.present?
        sub_fields
```
I assume it gets the subfields from the question ?

**Comment by RuslanHamidullin** [comment]
@jaleszek it's default
https://github.com/bitzesty/qae/blob/master/lib/qae_form_builder/address_question.rb#L41

**Comment by RuslanHamidullin** [comment]
I do not need to specify it in 3 places as  they same

**Comment by jaleszek** [comment]
Story doesn't cover changing it everywhere.

**Pull request merged**